airbyte-integrations/connectors/source-amazon-ads/integration_tests/future_state.json

```diff
@@ -0,0 +1 @@
+{ "display_report_stream": { "reportDate": "66660804" } }
```

Comment:
Which diff between this file and `abnormal_state.json`?

Reply:
This file is not used, will delete it

Reply:
Yes, thanks, updated PR